client/src/components/visibility/Visibility.tsx

```diff
@@ -87,7 +87,7 @@ export interface VisibilitiesInputProps {
   namespaceVisibility: Visibilities;
 
   /** Default value */
-  value: Visibilities | null;
+  value?: Visibilities;
```

Comment:
I think the original typing is more semantically correct:

```suggestion
  value: Visibilities | null;
```